examples/room-manager/src/plugins/fishjam.ts

```diff
@@ -1,24 +1,35 @@
+import fp from 'fastify-plugin';
```

Comment:
**nitpick:** This shortcut might be a bit misleading, it's often used to indicate functional programming API, for example, "lodash/fp" etc. 

And another thing is it clashes with fishjam. Is this `fastify-plugin` or `fishjam-plugin`.

I'd suggest to just import it as `fastifyPlugin`.

Reply:
this is as well the default approach in fastify documentation, so I think fastify users will not be confused